pymatgen/command_line/gulp_caller.py

```diff
@@ -1,5 +1,5 @@
 """Interface with command line GULP.
-http://projects.ivec.org
+https://gulp.curtin.edu.au/index.html
```

Comment:
This website update need confirmation: 
- Is the original url not accessible at all, or just temporarily, or just something with my network?
- Is the new website pointing to the same package?

Meanwhile it might be hard to automate the download process because the new website requires a form being filled.

Reply:
> Is the original url not accessible at all, or just temporarily, or just some issue with my network?

i also can't access http://projects.ivec.org. very likely it's permanently down.

> Is the new website pointing to the same package?

yep, that's the one

> Meanwhile it might be hard to automate the download process because the new website requires a form being filled.

i think it's fine to ask users to install manually. this is not core `pymatgen` functionality